packages/transformer-typescript/src/index.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+
+import { createTransformer, shouldProcess, MessageIssue } from 'pundle-api'
+import type { File } from 'pundle-api/types'
+
+// TODO: Implement watching and everything
+export default createTransformer(async function(config: Object, file: File) {
+  if (!shouldProcess(this.config.rootDirectory, file.filePath, config)) {
+    return null
+  }
+
+  let typescriptPath
+  try {
+    typescriptPath = await this.resolve('typescript')
+  } catch (_) {
+    throw new MessageIssue('Unable to find babel-core in project root', 'error')
```

Comment:
Why are you dynamically detecting typescript and throwing a custom error, rather than letting Node handle it?